client/store/potholes.js

```diff
@@ -41,21 +47,33 @@ export const createGotPotholeThunk = potholeId => {
       const { data: singlePothole } = await axios.get(
         `/api/potholes/${potholeId}`
       );
-      console.log(singlePothole);
       dispatch(createGotSinglePotholeAction(singlePothole));
     } catch (error) {
       console.error(error);
     }
   };
 };
 
+export const createUpdateStatusThunk = (pothole, potholeId) => {
+  return async dispatch => {
+    try {
+      const response = await axios.put(`/api/potholes/${potholeId}`, pothole);
```

Comment:
the information returned from axios is contained in the data key, correct? I don't think we need to be sending the entire response object